src/mongo/s/stats/joom_latency_histogram.h

```diff
@@ -29,12 +29,40 @@ const static StringMap<CommandName> StringToCommandNameMap = {
     {"distinct", CommandName::Distinct},
 };
 
+const static std::vector<uint64_t> exponentialBuckets(uint64_t min, uint64_t max, size_t count) {
+    std::vector<uint64_t> buckets;
+    buckets.reserve(count);
+    double step = std::exp(std::log(max/min) / static_cast<double>(count - 1));
+
+    uint64_t value = min;
+    for (size_t i = 0; i < count - 1; ++i) {
+        buckets.push_back(std::round(value));
```

Comment:
There is no point in rounding integer value.

Reply:
Now `value` is a double.